pkg/types/openstack/validation/machinepool.go

```diff
@@ -8,5 +8,17 @@ import (
 
 // ValidateMachinePool checks that the specified machine pool is valid.
 func ValidateMachinePool(p *openstack.MachinePool, fldPath *field.Path) field.ErrorList {
-	return field.ErrorList{}
+	allErrs := field.ErrorList{}
+
+	// Validate Volumes
+	if p.RootVolume != nil {
```

Comment:
based on https://github.com/openshift/installer/pull/2394/files#diff-f4a1bfbddde14b37dea44f2357757492R35-R40

both the values must be set when RootVolume is not-nil, so why do we have 
`p.RootVolume.Type == "" && p.RootVolume.Size > 0`
`p.RootVolume.Type != "" && p.RootVolume.Size <= 0`

why not 2 conditions 
`p.RootVolume.Type == ""` -> error
`p.RootVolume.Size <=0` -> error

Reply:
Done